### PR TITLE
Pull in upstream updates

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request pulls in upstream changes from [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic).

## 💭 Motivation and Context

I plan to make some changes to the code-gov-update deployment related to cisagov/cyhy_amis#279, so it makes sense to go ahead and update all the related repositories.

## 🧪 Testing

All pre-commit hooks pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,18 +110,18 @@ Once `pyenv` and `pyenv-virtualenv` are installed on your system, you
 can create and configure the Python virtual environment with these
 commands:
 
-```bash
+```console
 cd skeleton-generic
 pyenv virtualenv <python_version_to_use> skeleton-generic
 pyenv local skeleton-generic
```
Comment (mcdonnnj) [inline CONTRIBUTING.md]: This never got updated from the skeleton.

**Review by mcdonnnj — APPROVED** [review]
Thanks for bringing another one into the light 🔦 Please see my comment about some documentation that got missed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jsf9k) [reply CONTRIBUTING.md]: Sharp eyes!  Fixed in commit a6d2e5e.

**Review by dav3r — APPROVED** [review]
⛽

**Pull request merged**